### PR TITLE
docs: restructure app docs

### PR DESCRIPTION
**Pull request by vladitasev**

### CODE REVIEW

File: docs/2-advanced/08-using-i18n-for-apps.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ None of the code below implies or requires the usage of UI5 Web Components, and
 The `@ui5/webcomponents-base` package allows the usage of `i18n` functionality not just for UI5 Web Components,
```
Comment (nnaydenow) [inline docs/2-advanced/08-using-i18n-for-apps.md:6]: can we add front matter at the beginning of the file
```
---
title: i18n for Apps
---
```

File: docs/2-advanced/09-accessibility.md
Comment (nnaydenow) [inline docs/2-advanced/09-accessibility.md:1]: Same here:
---
title: Accessibility
---

**Review by nnaydenow — COMMENTED** [review]
same for `Right-To-Left (RTL) and Compact Mode` => `RTL & Compact Mode`

File: docs/2-advanced/08-using-i18n-for-apps.md (hunk 1 above)
Comment (vladitasev) [reply docs/2-advanced/08-using-i18n-for-apps.md:6]: sure

**Review by ilhan007 — APPROVED** [review]

**Comment by ilhan007** [comment]
Related to: https://github.com/SAP/ui5-webcomponents/issues/9190

**Pull request merged**